paper/paper.bib

```diff
@@ -314,7 +315,7 @@ @inproceedings{Bartman_et_al_2023
 }
 
 @inproceedings{PyPartMC_AMS2023,
-  title = {PyPartMC: a {P}ythonic Interface to a Particle-Resolved {M}onte-{C}arlo Aerosol Simulation Framework},
+  title = {PyPartMC: a Pythonic Interface to a Particle-Resolved Monte-Carlo Aerosol Simulation Framework},
```

Comment:
the paper.bib file should not be changes in this PR

Reply:
Thanks. I'm not sure why my PR's always seem to revert something in the paper... maybe time to start from a fresh install locally.